src/main/java/org/example/models/PlayerStatus.java

```diff
@@ -31,8 +35,9 @@ public WeaponModel getWeaponInHand() {
 
     public void setWeaponInHand(WeaponModel weapon, Weapons weaponType) {
```

Comment:
For better readability, I'd change the name to something like 'equipWeapon'. Usually only setters start with 'set'.